developers/submit-data.md

```diff
@@ -12,6 +12,16 @@ To submit data to Celestia, users submit blob transactions (`BlobTx`). Blob
 transactions contain two components, a standard Cosmos-SDK transaction called
 `MsgPayForBlobs` and one or more `Blob`s of data.
 
+## Maximum blob size
+
+The maximum total blob size in a transaction is just under **2 MiB
```

Comment:
Although this is technically the max blob size, I think we should provide an advisable maximum size of something like 80-90% of the max size to avoid timeouts on users submitting large blobs.

Reply:
This doesn't make sense to me yet WHY it happens. I know things would get congested if the entire block gets filled, but I wasn't aware it happens to individual blobs as well?

Reply:
if the whole block is filled with just 1 blob then there is no room for any other transactions, so it will most likely take longer to get it included, unless you really jack up the gas

Reply:
so i think encouraging people to not try to push the limit would help things get included without wasting gas on beating the lower fees out